ora_migrator--1.1.0.sql

```diff
@@ -51,6 +51,30 @@ $$DECLARE
          '  AND owner NOT IN (' || sys_schemas || E')'
       ')'', max_long ''%s'', readonly ''true'')';
 
+   table_comments text := E'CREATE FOREIGN TABLE %I.table_comments (\n'
+      '   schema     text NOT NULL,\n'
+      '   table_name text NOT NULL,\n'
+      '   table_comment text NOT NULL\n'
+      ') SERVER %I OPTIONS (table ''('
+         'SELECT owner,\n'
+         '       table_name\n'
```

Comment:
In a query that uses more than one table, please add the table alias to all column references (e.g., use `t.owner` rather than `owner`). I am surprised that Oracle does not give you an error message there.
The exception is if you join tables with the `USING` clause &mdash; then you don't need to use the table alias.